packages/components/src/components/icon/Usage.mdx

```diff
@@ -0,0 +1,78 @@
+
+import { Meta } from '@storybook/blocks';
+
+import * as IconStories from './icon.stories';
```

Comment:
here, the correct path is: './infineonIconStencil.stories';

Reply:
Thanks, i've fixed it :)